vitest.shared.mts

```diff
@@ -36,24 +36,7 @@ export const alias = [
 ];
 
 export default defineConfig({
-  plugins: [
-    react({
-      babel: {
-        plugins: [
-          [
-            '@mui/internal-babel-plugin-display-name',
-            {
-              allowedCallees: {
-                '@mui/x-internals/forwardRef': ['forwardRef'],
-              },
-            },
-          ],
-        ],
-        babelrc: false,
-        configFile: false,
-      },
-    }),
-  ],
+  plugins: [react()],
```

Comment:
This is the main change, everything else is just good to have. 

I've tracked it down to this change. I didn't test if it is babel or just our plugin. Removed all.